### PR TITLE
[KIECLOUD-538] - Reconciliation loop if set CPU limit to 1000m

### PR DESCRIPTION
**Pull request by spolti**
Signed-off-by: spolti <fspolti@redhat.com>

### CODE REVIEW

File: pkg/controller/kieapp/kieapp_controller.go [hunk 1]
```diff
@@ -95,6 +95,8 @@ func (reconciler *Reconciler) Reconcile(request reconcile.Request) (reconcile.Re
 		return reconcile.Result{}, err
 	}
 
+	// maybe add the formater here
+
```
Comment (tchughesiv) [inline pkg/controller/kieapp/kieapp_controller.go:99]: note still needed?

File: pkg/controller/kieapp/defaults/defaults_test.go [hunk 2]
```diff
@@ -1773,6 +1758,16 @@ func TestConstructServerObject(t *testing.T) {
...
+		//assert.Equal(t, "1e3", cr.Spec.Objects.Servers[0].Resources.Limits.Cpu().String())
+		//assert.Equal(t, "2Gi", cr.Spec.Objects.Servers[0].Resources.Limits.Memory().String())
+		//assert.Equal(t, "500", cr.Spec.Objects.Servers[0].Resources.Requests.Cpu().String())
+		//assert.Equal(t, "1", cr.Spec.Objects.Servers[0].Resources.Requests.Memory().String())
+		//
+		//assert.Equal(t, "1e3", cr.Status.Applied.Objects.Servers[0].Resources.Limits.Cpu().String())
+		//assert.Equal(t, "10", cr.Status.Applied.Objects.Servers[0].Resources.Limits.Memory().String())
+		//assert.Equal(t, "500", cr.Status.Applied.Objects.Servers[0].Resources.Requests.Cpu().String())
+		//assert.Equal(t, "1", cr.Status.Applied.Objects.Servers[0].Resources.Requests.Memory().String())
+
```
Comment (tchughesiv) [inline pkg/controller/kieapp/defaults/defaults_test.go:1770]: these still needed?

File: pkg/controller/kieapp/defaults/defaults.go [hunk 3]
```diff
@@ -295,6 +295,7 @@ func findCustomObjectByName(template api.CustomObject, objects []api.CustomObjec
 }
 
 func getEnvTemplate(cr *api.KieApp) (envTemplate api.EnvTemplate, err error) {
+	// TODO
```
Comment (tchughesiv) [inline pkg/controller/kieapp/defaults/defaults.go:298]: note still needed?

Comment (spolti) [reply pkg/controller/kieapp/defaults/defaults.go:298]: no, these are letfovers, forgot to remove them.

**Pull request merged**